pkg/config/config.go

```diff
@@ -372,23 +436,56 @@ func (s SystemConfig) getZoneConfigForID(id uint32) (ZoneConfig, error) {
 // ComputeSplitKey takes a start and end key and returns the first key at which
 // to split the span [start, end). Returns nil if no splits are required.
 //
-// Splits are required between user tables (i.e. /table/<id>) and at the start
-// of the system-config range (i.e. /table/0). The system-config range is
-// somewhat special in that it contains multiple SQL tables
-// (/table/0-/table/<max-system-config-desc>) which must be contained in a
-// single range.
+// Splits are required between user tables (i.e. /table/<id>), at the start
+// of the system-config tables (i.e. /table/0), and at certain points within the
+// system ranges that come before the system tables. The system-config range is
+// somewhat special in that it can contain multiple SQL tables
+// (/table/0-/table/<max-system-config-desc>) within a single range.
 func (s SystemConfig) ComputeSplitKey(startKey, endKey roachpb.RKey) roachpb.RKey {
-	systemConfigStart := roachpb.RKey(keys.TableDataMin)
-	if !systemConfigStart.Less(endKey) {
-		// endKey <= systemConfigStart: no required splits.
-		return nil
+	// Before dealing with splits necessitated by SQL tables, handle all of the
+	// static splits earlier in the keyspace. Note that this list must be kept in
+	// the proper order (ascending in the keyspace) for the logic below to work.
+	staticSplits := []struct {
```

Comment:
It was already pretty effectively tested by `TestComputeSplitKeySystemRanges`, but done.